app/default_overrides.go

```diff
@@ -75,12 +79,33 @@ func (stakingModule) DefaultGenesis(cdc codec.JSONCodec) json.RawMessage {
 	params := stakingtypes.DefaultParams()
 	params.UnbondingTime = appconsts.DefaultUnbondingTime
 	params.BondDenom = BondDenom
+	params.MinCommissionRate = sdk.NewDecWithPrec(5, 2) // 5%
 
 	return cdc.MustMarshalJSON(&stakingtypes.GenesisState{
 		Params: params,
 	})
 }
 
+// stakingModule wraps the x/staking module in order to overwrite specific
+// ModuleManager APIs.
+type slashingModule struct {
+	slashing.AppModuleBasic
+}
+
+// DefaultGenesis returns custom x/staking module genesis state.
+func (slashingModule) DefaultGenesis(cdc codec.JSONCodec) json.RawMessage {
+	params := slashingtypes.DefaultParams()
+	params.MinSignedPerWindow = sdk.NewDecWithPrec(5, 2) // 5%
+	params.SignedBlocksWindow = 15000
+	params.DowntimeJailDuration = time.Minute * 1
```

Comment:
https://github.com/celestiaorg/celestia-app/blob/4e62d2d3660e8ad5b6115348bed5aa6a57cf93fc/specs/src/specs/params.md?plain=1#L46 needs to be updated

Reply:
https://github.com/celestiaorg/celestia-app/pull/2417/commits/0d6fd942bd3b34d453a5ced73de456be9b264be2

updated! thanks for catching

Reply:
used 10 minutes for both values per sync discussion